platform-includes/getting-started-use/javascript.nestjs.mdx

```diff
@@ -73,12 +72,13 @@ import { SentryGlobalFilter } from "@sentry/nestjs/setup";
 })
 export class AppModule {}
 ```
+
 {/* TODO(v9): Remove this note. */}
 
 **Note:** If you have a NestJS + GraphQL application and you are using the `@sentry/nestjs` SDK version `8.38.0` or earlier, replace the `SentryGlobalFilter` with the `SentryGlobalGenericFilter`.
 In SDK versions `8.39.0` and above, the `SentryGlobalGenericFilter` is deprecated because the `SentryGlobalFilter` will handle GraphQL contexts automatically.
 
-By default, exceptions with status code 4xx are not sent to Sentry. If you still want to capture these exceptions, you can do so manually with `Sentry.captureException()`:
+If you have error filters for specific types of exceptions (for example `@Catch(HttpException)`, or any other `@Catch(...)` with arguments) and you want to capture errors caught by these filters, capture the errors in the `catch()` handler with `Sentry.captureException()`:
```

Comment:
In this case they would also need to remove our `SentryExceptionCaptured` decorator, otherwise unexpected errors get captured twice right?

Reply:
This is a tricky one to think through. In general, our dedupe integration will drop the second capture call, as long as the two capture calls happen consecutively (no capture call on another error in between).

I am not sure if we should tell people to remove their `SentryExceptionCaptured` decorators though. The two things serve different purposes. One is more fine grained the other more blanket.